### ISSUE TITLE
Print Screen Hotkey does not work anymore with Windows 11 KB502531

### ISSUE THREAD

**Issue by fratzi** [greenshot/greenshot#482]
Hi,

with Windows 11 KB5025310 (Insider Build) windows is using print-screen button for opening snipping tool.
With this, Greenshot can't map and overwrite anymore the printscreen button. There is also a error message at the first start of greenshot. 

My Greenshot Build is 1.2.10 build 6

**Comment by fratzi** [comment]
This was just for information - maybe GS could inform about the possibility to disable the new windows built in screenshot feature. Currently it's only faulting with an error message, that it can't bind print-screen button.

**Comment by jglathe** [comment]
Agree. However I thought about doing just this, on install? https://www.elevenforum.com/t/enable-or-disable-use-print-screen-key-to-open-screen-snipping-in-windows-11.520/

The registry key would look like this :

```
Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Control Panel\Keyboard]
"PrintScreenKeyForSnippingEnabled"=dword:00000000
```

I tried it out on my wdk2023 (arm64 😁) and it worked nicely. If the default PrtScr feature is disabled, first start after install doesn't have an issue either. 

![2023-04-15 20_22_17-Print Screen Hotkey does not work anymore with Windows 11 KB502531 · Issue #482 ](https://user-images.githubusercontent.com/33185912/232247359-bcd7417f-3c1e-4255-8b58-1e6d24f7195e.png)
![2023-04-15 20_23_34-Info über Windows](https://user-images.githubusercontent.com/33185912/232247361-1abd7e83-3dfc-424a-b249-788b12c5c5c3.png)

It may be interesting to note that the registry key actually isn't there first, but it will be changed according to the setting of the feature in windows settings. I re-confirmed it by setting and un-setting it via regedit, the setting changed in the dialog (after reopening it), too.

![2023-04-15 20_34_17-Einstellungen](https://user-images.githubusercontent.com/33185912/232247539-4d205be8-e98e-46c0-a649-f6f5b7e4c249.png)

**Comment by jglathe** [comment]
[Binaries for x86 and ARM64](https://github.com/jglathe/greenshot/releases/tag/1.3.280-arm64)

**Comment by Slesa** [comment]
Well, I have the "Use Printscreen" disabled, but it still does not work. This is so nasty... I wish I could keep Windows 10 :-/

**Comment by arkadiusz-koziel-nc** [comment]
Try to open regedit as an Administrator and create below entry  DWORD > (32-bit Value)
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\ Microsoft\TabletPC\DisableSnippingTool = 1
GS restart
Also Dropbox save screenshot option should be disabled.

**Comment by NitemareReal** [comment]
Disabling "Use the Print screen key to open Snipping Tool" worked for me.
I had to "re-save" GreenShot hotkeys settings and that's all!

**Comment by zeonos** [comment]
Just for those stumbling onto this, some logitech keyboards also rebind the screenshot key, this can be changed in logi options.
